bin/inis/LilyPad.ini

```diff
@@ -0,0 +1,203 @@
+[General Settings]


```

Comment:
These user .inis containing custom values/filepaths need to be removed for this PR, same applies for the other ones.

Reply:
You are commenting on previous commits, since then issues like unnecessary ini edits have been corrected.  If it would be easier I could squash all of this work down to one commit?

Reply:
Yes that would help navigation greatly. Try to squash what you can (but keep the commits small) or leave it for now. Your choice I guess.